.circleci/config.yml

```diff
@@ -1,30 +1,34 @@
 version: 2
 jobs:
-  build:
+  no_pic:
     working_directory: ~/dmd
     docker:
       - image: circleci/node:4.8.2
```

Comment:
We could replace this with a D Docker image that doesn't require `-fPIC`, e.g. https://github.com/wilzbach/dlang-docker/blob/master/circleci/dlang.docker built for Ubuntu 14.04, but this should be more than fine for now (it's the existing status quo after all).